src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -46,8 +62,18 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
         const paramsScript = this._getOptionsScript(initializationData, currencyCode);
         const buttonParams: ButtonsOptions = {
             style: buttonStyle,
-            onApprove: data => this._tokenizePayment(data, submitForm),
-            onClick: async (_, actions) => onValidate(actions.resolve, actions.reject),
+            onApprove: data => {
+                this._deinitializePollingTimer(this._pollingInterval);
```

Comment:
Not sure if causes any issue, but it is worthy to do that only for alternative method

Reply:
fixed